### PR TITLE
Add task to configure apt source list

### PR DESCRIPTION
**Pull request by SlothOfAnarchy**
This is needed for other architectures, like arm64

### CODE REVIEW

File: templates/sources.list.j2 [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#{{ansible_architecture}}
```
Comment (dasJ) [inline templates/sources.list.j2]: -

File: tasks/common_packages.yml [hunk 2]
```diff
@@ -1,4 +1,13 @@
 ---
+- name: Write the apt sources file
+  template:
+    src: sources.list.j2
+    dest: /etc/apt/sources.list
+    mode: 0644
+    owner: root
+    group: root
+
+
```
Comment (dasJ) [inline tasks/common_packages.yml]: -

File: templates/sources.list.j2 (hunk 1 above)
Comment (SlothOfAnarchy) [reply templates/sources.list.j2]: My bad, I meant `ansible_managed`.

File: templates/sources.list.j2 [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+deb http://security.ubuntu.com/ubuntu {{ ansible_distribution_release }}-security main universe
+{% else %}
+deb http://ports.ubuntu.com/ubuntu-ports {{ ansible_distribution_release }}-updates main universe
+{% endif %}
+
+{% if ansible_architecture=='x86_64' %}
+deb http://archive.ubuntu.com/ubuntu {{ ansible_distribution_release }}-updates main universe
+{% else %}
+deb http://ports.ubuntu.com/ubuntu-ports {{ ansible_distribution_release }}-security main universe
+{% endif %}
```
Comment (Mr-Pi) [inline templates/sources.list.j2]: Please use just one if else block, thanks.

File: tasks/common_packages.yml (hunk 2 above)
Comment (Mr-Pi) [reply tasks/common_packages.yml]: @dasJ where is the problem here?

Comment (SlothOfAnarchy) [reply tasks/common_packages.yml]: I think he is denouncing the unnecessary whitespace at the end of this file.

**Comment by SlothOfAnarchy** [comment]
Please rereview @dasJ @Mr-Pi

**Comment by dasJ** [comment]
Still blocked by @Mr-Pi

**Pull request merged**